master/buildbot/status/master.py

```diff
@@ -129,6 +129,12 @@ def getTitleURL(self):
     def getBuildbotURL(self):
         return self.master.config.buildbotURL
 
+    def getBuildbotURLAliases(self):
```

Comment:
so I need to change https://github.com/buildbot/buildbot/blob/master/master/buildbot/config.py#L194 to :8010?